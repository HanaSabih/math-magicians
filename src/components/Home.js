import { Container } from 'react-bootstrap';

const Home = () => (
  <Container className="  mt-5 homeContainer ">
    <h1 className="  mb-5  ">Welcome to Math magicians!</h1>

    <p>
      This is a simple calculator with memory functions similar to a small handheld calculator.
      Use this basic calculator online for math with addition,
      subtraction, division and multiplication.
    </p>
    <p>
      If it&apos;s your first time using a calculator,
      all the buttons and options can be a bit scary.
      But whether you&apos;re using a standard calculator or scientific calculator,
      the basics are pretty much the same.
      Once you learn what the buttons do and how to use them for different calculations,
      you&apos;ll be more than comfortable using your calculator whenever
      you need it—inside or outside of school!
    </p>
    <p>
      <strong>Add numbers</strong>
      {' '}
      with the &quot;+&quot; key. Hit the &quot;+&quot; button between
      any 2 numbers to add them.
      For example, to add 5 to 10, press &quot;5,&quot; &quot;+,&quot;
      and then &quot;10.&quot;
    </p>

    <p>
      <strong>Subtract numbers</strong>
      {' '}
      with the &quot;-&quot; key. Press the &quot;-&quot; button between
      any 2 numbers to subtract the second from the first.
      For example, hit &quot;7,&quot; &quot;-,&quot; and then &quot;5&quot; to
      subtract 5 from 7 and hit &quot;=&quot; to get an answer of &quot;2.&quot;
    </p>
    <p>
      <strong>Divide numbers</strong>
      {' '}
      &quot;&quot;

      or turn fractions into decimals with the &quot;÷&quot; or &quot;/&quot; key.
      For example, to divide 2 by 1, press &quot;2,&quot;
      &quot;÷&quot; and &quot;1&quot; and then &quot;=.&quot;
      To change the fraction 4/5 to a decimal, press &quot;4,&quot;
      &quot;/,&quot; &quot;5,&quot; and then &quot;=.&quot;
    </p>
    <p>
      <strong>Multiply numbers</strong>
      {' '}
      using the &quot;x&quot; or &quot;*&quot; key. For example,
      to multiply 6 by 5, press &quot;6,&quot; &quot;x,&quot; &quot;5,&quot;
      and then &quot;=.&quot; The final answer will read &quot;30.&quot;
    </p>
  </Container>
);

export default Home;
