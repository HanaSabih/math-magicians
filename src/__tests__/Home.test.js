import renderer from 'react-test-renderer';
import Home from '../components/Home';

it('renders correctly', () => {
  const testQuote = renderer.create(<Home />).toJSON();
  expect(testQuote).toMatchSnapshot();
});
