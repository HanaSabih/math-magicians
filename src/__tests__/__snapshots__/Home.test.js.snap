// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="  mt-5 homeContainer  container"
>
  <h1
    className="  mb-5  "
  >
    Welcome to our page!
  </h1>
  <p>
    This is a simple calculator with memory functions similar to a small handheld calculator. Use this basic calculator online for math with addition, subtraction, division and multiplication.
  </p>
  <p>
    If it's your first time using a calculator, all the buttons and options can be a bit scary. But whether you're using a standard calculator or scientific calculator, the basics are pretty much the same. Once you learn what the buttons do and how to use them for different calculations, you'll be more than comfortable using your calculator whenever you need it—inside or outside of school!
  </p>
  <p>
    <strong>
      Add numbers
    </strong>
     
    with the "+" key. Hit the "+" button between any 2 numbers to add them. For example, to add 5 to 10, press "5," "+," and then "10."
  </p>
  <p>
    <strong>
      Subtract numbers
    </strong>
     
    with the "-" key. Press the "-" button between any 2 numbers to subtract the second from the first. For example, hit "7," "-," and then "5" to subtract 5 from 7 and hit "=" to get an answer of "2."
  </p>
  <p>
    <strong>
      Divide numbers
    </strong>
     
    "" or turn fractions into decimals with the "÷" or "/" key. For example, to divide 2 by 1, press "2," "÷" and "1" and then "=." To change the fraction 4/5 to a decimal, press "4," "/," "5," and then "=."
  </p>
  <p>
    <strong>
      Multiply numbers
    </strong>
     
    using the "x" or "*" key. For example, to multiply 6 by 5, press "6," "x," "5," and then "=." The final answer will read "30."
  </p>
</div>
`;
